dev-libs/mongo-c-driver/mongo-c-driver-0.98.2.ebuild

```diff
@@ -25,6 +25,10 @@ DEPEND="${RDEPEND}
 
 DOCS=( NEWS README.rst TUTORIAL.md )
 
+# No tests on x86 because tests require dev-db/mongodb which don't support
+# x86 anymore (bug #645994)
+RESTRICT="x86? ( test )"
```

Comment:
I am not sure that this is enough. Due to recent changes in portage I think we have to package.use.mask "test" for x86 to get rid of mongodb dependency.

Reply:
@Whissi, seems like we need to change https://github.com/gentoo/gentoo/blob/master/profiles/arch/x86/package.use.mask#L63 and replace ">=dev-libs/mongo-c-driver-1.8.2 test" with "dev-libs/mongo-c-driver test". I remember you placed a use.mask, but I forgot it was just for a specific version.